### PR TITLE
style: adjust button line-height to support 2 lines of text in big bu…

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

[Different Button Height on Safari iPhone 13 ](https://dfinity.atlassian.net/browse/L2-709)

# Changes

- line-height is adjusted to support 2 lines within normal height.

# Tests

## before
image

## after
image

### small buttons should look the same
image

### CODE REVIEW

**Review by lmuntaner — APPROVED** [review]
LGTM.

Also in Modals it appears well I asume, right?

File: frontend/svelte/src/lib/themes/button.scss [hunk 1]
```diff
@@ -51,6 +51,7 @@ button {
     min-height: var(--button-min-height);
 
     font-size: var(--font-size-small);
+    line-height: 1.25;
```
Comment (peterpeterparker) [inline frontend/svelte/src/lib/themes/button.scss:54]: `--line-height-standard` is 1.3, that wouldn't work? just would rather like not to have an "hardcoded" value

**Review by peterpeterparker — APPROVED** [review]
Thanks

**Comment by mstrasinskis** [comment]
> Also in Modals it appears well I asume, right?
@lmuntaner I didn't find any issue.

**Pull request merged**